### PR TITLE
Update Set-SPOTenant.md

### PR DESCRIPTION
**Pull request by SanjoyanM**
### -EnableSensitivityLabelforPDF

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -1825,6 +1826,31 @@ False (default) - Disables the policy.
...
+Applicable: SharePoint Online
+Required: False
+Position: Named
+Default value: True
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+### -EnableSensitivityLabelforPDF
+
+Tenant admin can turn on support for PDFs with sensitivity labels for the following scenarios:
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1840]:
```suggestion
Allows tenant admins to turn on support for PDFs with sensitivity labels for the following scenarios:
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 2]
```diff
@@ -1825,6 +1826,31 @@ False (default) - Disables the policy.
...
+
+1. Applying a sensitivity label in Office on the web
+2. Uploading a labeled document, and then extracting and displaying that sensitivity label
+3. Search, eDiscovery, and data loss prevention
+4. Auto-labeling policies and default sensitivity labels for SharePoint document libraries
+
+The valid values are:
+True - Enables the support for PDF.
+False (default) - Disables the support for PDF.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1849]:
```suggestion
True - Enables support for PDFs.

False (default) - Disables support for PDFs.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @SanjoyanM, kindly commit the suggested changes. Many thanks.

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 3]
```diff
@@ -1825,6 +1826,32 @@ False (default) - Disables the policy.
...
+
+- Applying a sensitivity label in Office for the web.
+- Uploading a labeled document, and then extracting and displaying that sensitivity label.
+- Search, eDiscovery, and data loss prevention.
+- Auto-labeling policies and default sensitivity labels for SharePoint document libraries.
+
+The valid values are:
+True - Enables support for PDFs.
+
+False (default) - Disables support for PDFs.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1850]:
```suggestion

- True - Enables support for PDFs.
- False (default) - Disables support for PDFs.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @SanjoyanM, one more, please.

**Comment by SanjoyanM** [comment]
@JohanFreelancer9 : I have accepted ALL comments. Please help merge this to main. My updates are important for customers showcasing new capabilities. Thanks in advance.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JohanFreelancer9** [comment]
> @JohanFreelancer9 : I have accepted ALL comments. Please help merge this to main. My updates are important for customers showcasing new capabilities. Thanks in advance.

@yogkumgit, can you please help? Thanks.

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 4]
```diff
@@ -1825,6 +1826,32 @@ False (default) - Disables the policy.
...
+
+- Applying a sensitivity label in Office for the web.
+- Uploading a labeled document, and then extracting and displaying that sensitivity label.
+- Search, eDiscovery, and data loss prevention.
+- Auto-labeling policies and default sensitivity labels for SharePoint document libraries.
+
+The valid values are:
+
+- True - Enables support for PDFs.
+- False (default) - Disables support for PDFs.
```
[the remark this answers is not in the document]
Comment (SanjoyanM) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1850]: I have fixed default value now in yaml.

[the remark this answers is not in the document]
Comment (get-itips) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1850]: It is still saying true for me

image

[the remark this answers is not in the document]
Comment (SanjoyanM) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1850]: My bad. Should reflect now.

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**